debian/control

```diff
@@ -32,6 +32,15 @@ Vcs-Browser: https://github.com/canonical/ubuntu-pro-client
 Rules-Requires-Root: no
 
 Package: ubuntu-advantage-tools
+Depends: ubuntu-pro-client, ${misc:Depends}
```

Comment:
We probably don't need `${misc:Depends}` here (but it's also not harmful in any way).

Reply:
I removed it but then lintian gives ` debhelper-but-no-misc-depends` as a warning so I'll add it back